website/docs/docs/introduction.md

```diff
@@ -3,188 +3,65 @@ title: "What is dbt?"
 id: "introduction"
 ---
 
-dbt enables analytics engineers to transform data in their warehouses by simply writing select statements. dbt handles turning these select statements into <Term id="table">tables</Term> and <Term id="view">views</Term>.
+## About dbt
 
-dbt does the `T` in <Term id="elt" /> (Extract, Load, Transform) processes – it doesn’t extract or load data, but it’s extremely good at transforming data that’s already loaded into your warehouse.
+<Snippet src="what-is-dbt-intro" />
 
-The role of dbt within a modern data stack is discussed in more detail [here](https://blog.fishtownanalytics.com/what-exactly-is-dbt-47ba57309068).
+dbt compiles and runs your analytics code against your data platform, enabling you and your team to collaborate on a single source of truth for metrics, insights, and business definitions. This single source of truth, combined with the ability to define tests for your data, reduces errors when logic changes, and alerts you when issues arise.
```

Comment:
Is "source of truth" a phrase that can be universally understood?  It's very impactful in English!

Reply:
Good question and I am not sure.

Reply:
I'm going to leave this for now, but we should follow up on this.